frontend/src/components/WindSettings.jsx

```diff
@@ -19,390 +20,159 @@ const useStyles = makeStyles((theme) => ({
-                                value={envConf.Wind.Force}
-                                inputProps={{ min: 0 }}
-                                fullWidth
-                            />
-                        </Tooltip>
-                    </Grid>
-                </Grid>
+const StyledSelect = styled(Select)(({ theme }) => ({
+    backgroundColor: '#F5F5DC',
+    '& .MuiInputBase-input': {
+        padding: '6px 8px',
+        height: '1em',
+    }
+}));
 
-            {(selectedWindType === "Turbulent Wind" || selectedWindType === "Wind Shear")  && (
-                    <Grid item container alignItems="center" direction="row">
-                        <Grid item xs={4}>
-                            <InputLabel id="fluctuation-label" sx={{ marginRight: 2, flexShrink: 0, color: '#F5F5DC', width: '200px' }}>
-                                Fluctuation %
-                            </InputLabel>
-                        </Grid>
-                        <Grid item xs={6}>
-                            <Tooltip title="Enter Fluctuation %" placement='bottom'>
-                                <TextField id="Fluctuation %" fullWidth
-                                    variant="outlined" 
-                                    type="number"
-                                    sx = {{
-                                        backgroundColor: '#F5F5DC',
-                                        '& .MuiOutlinedInput-root': {
-                                            '& .MuiInputBase-input': {
-                                                padding: '6px 8px',
-                                            },
-                                        },
-                                    }}
-                                    onChange={handleFLuctuationChange} 
-                                    value={fluctuationPercentage} 
-                                    inputProps={{ min: 0, 
-                                        max: 100, 
-                                        step: 0.1 
-                                    }} 
-                                />
-                            </Tooltip>
-                        </Grid>
-                    </Grid>
-            )}
+const WindDirection = [
+    { value: 'N', id: 5 },
+    { value: 'S', id: 6 },
+    { value: 'E', id: 7 },
+    { value: 'W', id: 8 },
+    { value: 'NE', id: 1 },
+    { value: 'SE', id: 2 },
+    { value: 'SW', id: 3 },
+    { value: 'NW', id: 4 }
+];
+
+const WindType = [
+    { value: "Constant Wind", id: 1 },
+    { value: "Turbulent Wind", id: 2 },
+];
+
+const WindSettings = ({
+    envConf, handleWindTypeChange, handleDirection, handleWindChange, handleFLuctuationChange,
+    selectedWindType, fluctuationPercentage, windShears,
+    setWindBlockData, deleteWindBlock
+}) => {
+    const classes = useStyles();
+    const [windBlocks, setWindBlocks] = useState(() => {
+        const storedWindBlocks = localStorage.getItem('windBlocks');
```

Comment:
@syedomair0 It's not a best idea to use localStorage here, because it stores values in the browser. I see that the Windblock data persists even after I refresh the webpage. 
We just have to store it in the internal state of the component.

Reply:
@syedomair0 I would suggest to remove the localStorage and move the state `const [windBlocks, setWindBlocks] = useState([])` into its parent component `EnvironmentConfiguration` and pass down this state as props to child component `WindSettings`.
Also make sure to sync the **Wind** property of **envConf** in `EnvironmentConfiguration` file with the latest windBlocks data. Essentially you have to call setEnvConf(updatedWindBlocks) within setWindBlocks(updatedWindBlocks).

Reply:
ah yes I was thinking that too... am working on it right now. will be fixed soon.